.bazelrc

```diff
@@ -6,10 +6,12 @@ run --show_result=0 --show_progress=no --incompatible_strict_action_env
 # what is defined in this section will be applied when bazel is invoked like this: bazel ... --config=rbe ...
 build:rbe --project_id=grakn-dev
 build:rbe --remote_instance_name=projects/grakn-dev/instances/default_instance
-build:rbe --remote_cache=remotebuildexecution.googleapis.com
-build:rbe --remote_executor=remotebuildexecution.googleapis.com
-build:rbe --bes_backend="buildeventservice.googleapis.com"
-build:rbe --bes_results_url="https://source.cloud.google.com/results/invocations/"
+build:rbe --remote_cache=cloud.buildbuddy.io
+build:rbe --remote_executor=cloud.buildbuddy.io
+build:rbe --bes_backend=cloud.buildbuddy.io
+build:rbe --bes_results_url=https://app.buildbuddy.io/invocation/
+build:rbe --tls_client_certificate=/home/circleci/.credentials/buildbuddy-cert.pem
+build:rbe --tls_client_key=/home/circleci/.credentials/buildbuddy-key.pem
```

Comment:
Can we change this path to be relative, @vmax ? eg. `.credentials/buildbuddy-cert.pem`

Reply:
@lolski we'd have to switch `checkout` and `install-bazel-linux` steps (you cannot `checkout` into non-empty directory) as well as modify `@graknlabs_dependencies`